index.html

```diff
@@ -510,14 +502,12 @@ <h3>Use Cases and Requirements</h3>
 <a>Issuers</a> can issue revocable <a>verifiable credentials</a>.
           </li>
           <li>
-The processes of cryptographically protecting <a>credentials</a> and
-<a>presentations</a>, and verifying <a>verifiable credentials</a> and
-<a>verifiable presentations</a>, have to be deterministic, bi-directional, and
-lossless. Any verification of a <a>verifiable credential</a> or
-<a>verifiable presentation</a> has to be transformable to the generic data model
-defined in this document in a deterministic process, such that the resulting
-<a>credential</a> or <a>presentation</a> is semantically and syntactically
-equivalent to the original construct, so that it can be processed in an
+The processes of cryptographically protecting and verifying
+<a>verifiable credentials</a> and <a>verifiable presentations</a> have to be
+deterministic, bi-directional, and lossless. Any
+<a>verifiable credential</a> or <a>verifiable presentation</a> has to be
+transformable to the JSON-LD data model defined in this document via a
+deterministic process so that its verification can be processed in an
```

Comment:
@brentzundel I am still struggling with this part. I know you didn't create this text, you are just trying to clean it up.... but the whole paragraph is problematic.

Many digital signatures are *not* deterministic, some datetime conversions are always lossy....

DataIntegrityProofs can be lossy ( order issues, context integrity issues, etc...)... Even assuming this paragraph was meant to only apply to data integrity proofs, I am not sure it is accurate.

Reply:
I think there a numerous problems with the whole use cases section, which is non-normative but feels very official. I've opened #1169 to have a conversation about removing it.